src/app/components/footer/footer.component.css

```diff
@@ -0,0 +1,3 @@
+#bulma-pic {
```

Comment:
In our client app, we do not use id's for styling. We use id's as selectors for e2e (end-to-end) testing. Change id's to classes.